geometry/optimization/spectrahedron.cc

```diff
@@ -77,8 +77,18 @@ std::unique_ptr<ConvexSet> Spectrahedron::DoClone() const {
 }
 
 bool Spectrahedron::DoIsBounded() const {
-  throw std::runtime_error(
-      "Spectrahedron::IsBounded() is not implemented yet.");
+  // CSDP cannot handle an unconstrained spectrahedron (i.e. a set whose only
```

Comment:
I'm not familiar with CSDP, but this seems strange? I would imagine that the important distinction is between bounded and unbounded, rather than constrained and unconstrained. For example, does CSDP accept something like `X >= 0` and `X[0,0] == 1`? (This would be constrained but unbounded.)

Reply:
There's an error in either CSDP or the Drake interface to CSDP. It will give there error:
```
Problem must have at least one constraint.
unknown file: Failure
C++ exception with description "CsdpSolver: the CSDP library exited via a fatal exception" thrown in the test body.
```